values.yaml

```diff
@@ -195,6 +195,63 @@ server:
   # -- Overrides the full name of the helm chart of the server component
   fullnameOverride: ""
 
+  # -- Add extra containers to the server component
+  extraContainers: []
+  # Example from https://litestream.io/guides/kubernetes/#litestream-application-container
```

Comment:
Imho these examples should be put somewhere in the docs not here

Reply:
@lafriks  This stuff has been removed in 0d9c83407d3a3d0bf8636f9e0aef3d7a69485308

Reply:
but some how it still shows up in PR diff 🤔